src/presentation/exercise_controller.spec.ts

```diff
@@ -1,12 +1,9 @@
 import fastify from 'fastify';
 import { mock } from 'jest-mock-extended';
 import { exerciseController } from './exercise_controller';
-import { Exercise, IExerciseRepository } from '../domain';
+import { Exercise, IExerciseRepository, Query } from '../domain';
 import { v4 as uuidv4 } from 'uuid';
-import {
-  IGetAnExerciseResponse,
-  IGetAllExercisesResponse,
-} from '../application';
+import { IGetAnExerciseResponse } from '../application';
```

Comment:
Måske vi skal overveje en code convention ift. absolutte stier til imports før relative stier til imports? hvad tænker du?

Reply:
https://github.com/OnlineTrainingPlatform/Exercises/pull/6#discussion_r1018989579

Reply:
Umiddelbart så synes jeg at absolutte imports (altså fra node_modules skal være i starten) og relative til sidst. Men det er non blocking for PR nu.